torch/testing/_internal/compilable_quantized_tensor.py

```diff
@@ -0,0 +1,366 @@
+"""
+Following is a example for a simple dtype implemented with tensor subclass
+it shows
+    * the basic structure of a new dtype tensor subclass (__new__, __init__, __tensor_flatten__, __tensor_unflatten__)
+    * two types of dispatch that people can overwrite (__torch_function__, __torch_dispatch__)
+    * how to abstract away packing format with layout
```

Comment:
hmm I'm not too sure what this means

Reply:
oh please take a look at https://github.com/pytorch/ao/issues/391 I can link it here as well